agent/kubernetes/service.go

```diff
@@ -18,6 +18,24 @@ type CostService struct {
 	influxdbClient client.Client
 }
 
+// PodCostComponents struct to hold all the cost information for a single pod
+type PodCostComponents struct {
+	totalCost        decimal.Decimal
```

Comment:
Can we either add the "cost" suffix to each field or remove it from `totalCost`?

Reply:
Will remove.